src/test/java/roomescape/integration/api/ThemeApiTest.java

```diff
@@ -0,0 +1,75 @@
+package roomescape.integration.api;
+
+import static org.hamcrest.Matchers.*;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import roomescape.common.BaseTest;
+
+class ThemeApiTest extends BaseTest {
+
+    private Map<String, String> createThemeRequest;
+
+    @BeforeEach
+    void setUp() {
+        createThemeRequest = Map.of(
+                "name", "공포의 방탈출",
+                "description", "무서운 분위기 속에서 탈출",
+                "thumbnail", "https://example.com/horror.jpg"
+        );
+    }
```

Comment:
```suggestion
    private final Map<String, String> createThemeRequest = Map.of(
                "name", "공포의 방탈출",
                "description", "무서운 분위기 속에서 탈출",
                "thumbnail", "https://example.com/horror.jpg"
        );
```

junit5부터는 기본 라이프 사이클이 메서드당이기 때문에 setUp을 만들 필요가 없습니다.

https://junit.org/junit5/docs/current/user-guide/#writing-tests-test-instance-lifecycle

Reply:
오 기본적으로 매번 인스턴스를 새로 생성하기 때문에 setup이 필요 없군요. 생명주기를 클래스 단위로 가져가고 싶다면 `@TestInstance(TestInstance.Lifecycle.PER_CLASS)`를 설정해 줘야 겠군요!